src/lib/index/wb_cache.cpp

```diff
@@ -275,10 +300,9 @@ void IndexWBCache::get_next_bufs(IndexCPContext* cp_ctx, uint32_t max_count, std
     }
 }
 
-void IndexWBCache::get_next_bufs_internal(IndexCPContext* cp_ctx, uint32_t max_count, IndexBuffer* prev_flushed_buf,
+void IndexWBCache::get_next_bufs_internal(IndexCPContext* cp_ctx, uint32_t max_count, IndexBufferPtr prev_flushed_buf,
```

Comment:
here too..

Reply:
Couldnt make it here, as we pass nullptr too